bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -610,7 +610,7 @@ public int getExplictLacInterval() {
     /**
      * Set the interval to check the need for sending an explicit LAC.
      * @param interval
-     *        Number of seconds between checking the need for sending an explict LAC.
+     *        Number of milli seconds between checking the need for sending an explict LAC.
      * @return Client configuration.
      */
     public ClientConfiguration setExplictLacInterval(int interval) {
```

Comment:
@eolivelli thanks for pointing that out.  I am still coming up to speed on all of BK, so if `setExplictLacInterval` has not been in an official release yet, then I am fine with just changing it.